File: Data Structures & Algorithms/duplicate-integer/submission-1.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        Set<Integer> num = new HashSet<Integer>();
        for(int i=0;i<nums.length;i++){
            boolean isAdded = num.add(nums[i]);
            if(!isAdded){
                return true;
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/rearrange-array-elements-by-sign/submission-1.py
class Solution:
    def rearrangeArray(self, nums: List[int]) -> List[int]:
        pos=[]
        neg=[]
        for num in nums:
            if num<=0:
                neg.insert(0,num)
            else:
                pos.insert(0,num)
        ret=[]
        for i in range(len(nums)//2):
            ret.append(pos.pop())
            ret.append(neg.pop())
        return ret